test/unit/modules/blocks/process.js

```diff
@@ -32,20 +32,37 @@ describe('blocks/process', function () {
 			node.constants.rewards.offset = 150;
 			// wait for mem_accounts to be populated
 			node.initApplication(function (err, __scope) {
-				setTimeout(function () {
-					scope = __scope;
-					accounts = __scope.modules.accounts;
-					blocksProcess = __scope.modules.blocks.process;
-					blocksVerify = __scope.modules.blocks.verify;
-					blockLogic = __scope.logic.block;
-					blocks = __scope.modules.blocks;
-					db = __scope.db;
-					done(err);
-				}, 5000);
+				scope = __scope;
+				accounts = __scope.modules.accounts;
+				blocksProcess = __scope.modules.blocks.process;
+				blocksVerify = __scope.modules.blocks.verify;
+				blockLogic = __scope.logic.block;
+				blocks = __scope.modules.blocks;
+				db = __scope.db;
+				done(err);
 			}, {db: db});
 		});
 	});
 
+	after(function (done) {
+		async.every([
+			'blocks where height > 1',
+			'trs where "blockId" != \'6524861224470851795\'',
+			'mem_accounts where address in (\'2737453412992791987L\', \'2896019180726908125L\')',
+			'forks_stat',
+			'votes where "transactionId" = \'17502993173215211070\''
+		], function (table, seriesCb) {
```

Comment:
table or query? What is happening here?

Reply:
and do we need these queries since we already have a global `after` which calls `dbSandbox.destroy`?